src/fediblockhole/__init__.py

```diff
@@ -183,7 +183,8 @@ def fetch_from_instances(sources: dict,
 
 def merge_blocklists(blocklists: list[Blocklist], mergeplan: str='max',
     threshold: int=0,
-    threshold_type: str='count') -> Blocklist:
+    threshold_type: str='count',
+    save_block_audit_file: str='') -> Blocklist:
```

Comment:
Why does this string default to empty? Wouldn't a `None` be a better indicator that no audit file is needed?
I'd be worried I'd accidentally clobber some other file in the working directory by opening an empty string for writing or something.